I-D-L2NM/draft-ietf-opsawg-l2nm.xml

```diff
@@ -657,15 +659,16 @@
         target="vpn-service_tree"></xref>.</t>
 
         <figure align="center" anchor="vpn-service_tree">
-          <artwork align="left"><![CDATA[     +--rw vpn-services
+          <artwork align="left"><![CDATA[     
+     +--rw vpn-services
         +--rw vpn-service* [vpn-id]
-           +--rw vpn-id                        vpn-common:vpn-id
+           +--rw vpn-id                        vpn-id
```

Comment:
The one with vpn-common prefix is correct

Reply:
The vpn-id is defined in the grouping vpn-description and we are using it on this part of the model. This is why this values does not have the prefix. 

````
list vpn-service {
        key "vpn-id";
        description
          "Container of port configurations";
        uses vpn-common:vpn-description;
````

I'll correct it manually.